views/job-roles/edit.html

```diff
@@ -0,0 +1,44 @@
+{% extends "templates/master.html" %}
```

Comment:
Could you go through and add the default bootstrap styling to each component? That should make the form look better